upmc-enterprises/cda2fhir
# Feature/upmc 249 250 allergy fields

#### What does this PR do?
Adds two Allergy Mappings:
- AllergyConcernAct.Author -> FHIR AllergyIntolerance.recorder 
- AllergyConcernAct.participant.participantRole.playingEntity.name -> AllergyIntolerance.code.text

This Also removed AllergyIntollerance.lastOccuance.
#### Related JIRA tickets:

- [UPMCFHIR-249](https://jira.amida.com/browse/UPMCFHIR-249)
- [UPMCFHIR-249](https://jira.amida.com/browse/UPMCFHIR-250)

#### How should this be manually tested?

Run documents through the transformer, print the output bundles, and check for substances in the output AllergyIntolerance.code.text fields.

#### Background/Context:

#### New JIRA tickets for clinical review required?
